slf4j-simple/src/main/java/org/slf4j/impl/SimpleLogger.java

```diff
@@ -126,7 +126,7 @@
     private static SimpleLogListener PRE_LOG_LISTENER = null;
     private static SimpleLogListener LOG_LISTENER = null;
 
-    private static int DEFAULT_LOG_LEVEL = LOG_LEVEL_INFO;
+    private static volatile int DEFAULT_LOG_LEVEL = LOG_LEVEL_INFO;
```

Comment:
I think this is not needed, as long as the static field is always modified inside a `synchronized` block (which seems to be the case here).

Reply:
After discussion with Steve, I admit that this is actually needed (the synchronisation does not cover all accesses to this field).